pdf_report/pdf_report.py

```diff
@@ -0,0 +1,34 @@
+import logging
+from dotenv import load_dotenv
+
+from transform_pdf import transform_pdf
+from load_pdf import load_pdf
+
+REPORT_FILENAME = 'performance_report.pdf'
+
+
+def main(_event, _context):
+    """ Lambda handler to run the PDF report pipeline. """
+
+    logging.basicConfig(level=logging.INFO,
```

Comment:
This should be changed to `    logging.getLogger().setLevel(logging.INFO)` for logs to show on AWS

Reply:
OH MY GOSH REALLY WOW

Reply:
YEAH I KNOW IT'S CRAZZZYYY